### DISCUSSION TITLE
QFieldCloud issue

### DISCUSSION THREAD

**Discussion by dto** [opengisch/QField#4080] [Bugs]
Forgive me if this is the wrong location to post this, I can't seem to find contact information for the folks involved with QFieldCloud. I have accidentally discovered a problem with the QFieldCloud website that you should definitely know about, but in the nature of the case is not appropriate for me to post details in a public forum. Can you please provide an email address for someone on the QFieldCloud team so that I can tell you what the problem is?

**Comment by ZsanettMed** [comment]
Hi @dto 
sure, please send us a mail to: beta-transition@qfield.cloud

**Reply by dto** [reply]
Ok, email sent! Thank you.

**Reply by ZsanettMed** [reply]
Thank you very much!
